src/puppetlabs/puppetdb/http/aggregate_event_counts.clj

```diff
@@ -23,9 +23,10 @@
                     query-options (-> {:counts_filter counts_filter'
                                        :count_by count_by}
                                       (merge distinct-options))]
-                (log/warn
-                  (str "The aggregate-event-counts endpoint is experimental"
-                       " and may be altered or removed in the future."))
+                (when-not (= "puppetdb" (:product-name globals))
```

Comment:
I think you're logic is inverted here. The product-name is *not* "puppetdb", meaning it's pe-puppetdb. So when pe-puppetdb, log the warning below, right? We only want this warning on FOSS.